interfaces/cos_agent/v0/charms.yaml

```diff
@@ -0,0 +1,5 @@
+providers:
+  - zookeeper
```

Comment:
this is the wrong format.
It should be:
```
providers:
  - name: zookeeper
    url: http:github.com/canonical/zookeeper-k8s-operator-or-whatever
```

use https://github.com/canonical/charm-relation-interfaces/blob/main/interfaces/ingress/v0/charms.yaml as template